CloneVkontakte/CloneVkontakte/GallaryCollectionViewController.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  GallaryCollectionViewController.swift
+//  CloneVkontakte
+//
+//  Created by Boris on 05.08.2021.
+//
+
+import UIKit
+
+private let reuseIdentifier = "imageCell"
```

Comment:
```suggestion
private let reuseIdentifier = "imageCell"
```
внутрь контролера переносим

Reply:
Поправил

Reply:
Если что, это автоматически создаётся при создании класса Коллекции ВьюКотроллера)))